openstudio-extension.gemspec

```diff
@@ -22,14 +27,15 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'bundler', '~> 1.9'
-  spec.add_dependency 'json_pure', '2.2.0'
+  spec.required_ruby_version = '>= 2.5'
```

Comment:
Should this be a pessimistic version constraint instead to lock at >= 2.5 and < 2.6? I know any 2.5.x works with openstudio at least. Not sure 2.6.x would work

Reply:
`~2.5.0` would lock at any 2.5.x

Reply:
You are right, this should be 2.5.0 to lock the 2.5.x series. I’ll update.